benchmarks/shootout/test_cleanup.sh

```diff
@@ -0,0 +1,4 @@
+# clean iwasms
+sudo rm -r ${WAMR_DIR}/product-mini/platforms/linux-sgx-1
+sudo rm -r ${WAMR_DIR}/product-mini/platforms/linux-sgx-2
+sudo rm -r ${WAMR_DIR}/product-mini/platforms/linux-sgx-3
```

Comment:
remove `sudo`

Reply:
`sudo` can't be removed if we want to delete those folders, because there are some build files that the user doesn't have access to write. The same rationale with `sudo` before

Reply:
Seems that you use `sudo` to create the folders? If wamr is cloned without `sudo`, we should be able to create/remove folders inside it withoult `sudo`

Reply:
I figure out why I add `sudo` in the first place. There are two write protected file in fast jit:
```shell
rm: remove write-protected regular file '/home/tl/TL/clion_projects/wasm-micro-runtime/product-mini/platforms/linux-sgx-fast-jit/build/_deps/asmjit-src/.git/objects/pack/pack-40d539042c7517a7db470d693740af1a8cc214b2.idx'? y
rm: remove write-protected regular file '/home/tl/TL/clion_projects/wasm-micro-runtime/product-mini/platforms/linux-sgx-fast-jit/build/_deps/asmjit-src/.git/objects/pack/pack-40d539042c7517a7db470d693740af1a8cc214b2.pack'? y
```
For now, I leave them be, and the user has to input 'y', any suggestions?

Reply:
It is a little strange, could you try `rm -fr` instead of `rm -r` to remove the folder?